test/tests/functional/pbs_multi_sched.py

```diff
@@ -700,16 +699,23 @@ def test_fairshare_usage(self):
         self.server.expect(JOB, {'job_state': 'Q'}, id=sc1_jid2)
         self.server.manager(MGR_CMD_SET, SCHED, {'scheduling': 'True'},
```

Comment:
When you will delete sc1_jid1, a sched cycle will be kicked automatically. So, kicking another one here is redundant now, so please remove this.

Reply:
Thanks @agrawalravi90 for the review, i tried your solution but couldn't get the expected output.
As per my understanding, need to trigger the scheduling cycle when job should be in 'R' state to update the fairshare usage file correctly.

Reply:
ah, ok, i think you are right, the fairshare usage won't get updated for sc1_jid1 's user as we will deleted the job and then kick the sched cycle. My bad, I take my comments back.